qp_woltka/util.py

```diff
@@ -121,8 +141,12 @@ def demux(input_, output):
             if current_fp is not None:
                 current_fp.close()
 
-            current_fp = open(output + sep + fname + ext, mode)
+            sample = search_by_filename(
+                fname.decode('utf8'), lookup).encode('ascii')
+            fullname = output + sep + sample + ext
+            current_fp = open(fullname, mode)
             current_fname = fname
+            print(fullname.decode('ascii'))
```

Comment:
Is this print needed?

Reply:
Yes, that's how we keep track of the samples/files that were processed.